### PR TITLE
Test modified may 24

### PR DESCRIPTION
**Pull request by manjunath-grl**
Summary: 
Modified tests
TC-BOOL-2.1
TC-FLW-2.2
TC-MC-1.5
TC-MC-1.7
TC-MC-3.2
TC-MC-3.4
TC-OCC-2.1
TC-PCC-2.1
TC-PS-1.1
TC-RH-2.2
TC-SC-4.2
TC-SWTCH-2.1
TC-SWTCH-2.2
TC-TM-2.2
TC-TSTAT-2.2
TC-WNCV-2.4
TC-WNCV-2.5
TC-WNCV-3.4
TC-WNCV-3.5
TC-WNCV-4.3
Added auto generated files

Attached execution logs:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/8771271/Execution_log.txt)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_FLW_2_2.yaml [hunk 1]
```diff
@@ -32,15 +32,23 @@ tests:
       command: "readAttribute"
       attribute: "MeasuredValue"
       response:
+          saveAs: ValueBeforeChange
           constraints:
               type: uint16
 
     - label: "operate on DUT to change the flow significantly"
-      disabled: true
+      PICS: MANUAL_TEMPERATURE_CHANGE
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_FLW_2_2.yaml:40]: What does this have to do with MANUAL_TEMPERATURE_CHANGE?

File: src/app/tests/suites/certification/Test_TC_RH_2_2.yaml [hunk 2]
```diff
@@ -52,15 +52,22 @@ tests:
...
           constraints:
               type: int16
               minValue: 0
               maxValue: 10000
 
-    #Disabled as UI prompts required for user action
     - label: "Operate on device to change the relative humidity significantly"
-      disabled: true
-      PICS: MANUAL_RELATIVEHUMIDITY_CHANGE
+      PICS: MANUAL_TEMPERATURE_CHANGE
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_RH_2_2.yaml:62]: Again, what does this have to do with MANUAL_TEMPERATURE_CHANGE?

File: src/app/tests/suites/certification/Test_TC_SC_4_2.yaml [hunk 3]
```diff
@@ -85,7 +85,6 @@ tests:
 
     - label: "Check Hostname"
       # On macOS the hostname is the device name and because of it this test is disabled for now.
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_SC_4_2.yaml:87]: Why is this comment still here?

File: src/app/tests/suites/certification/Test_TC_SC_4_2.yaml [hunk 4]
```diff
@@ -100,8 +99,7 @@ tests:
...
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_SC_4_2.yaml:101]: Why is this comment still here?

File: src/app/tests/suites/certification/Test_TC_FLW_2_2.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_FLW_2_2.yaml:40]: i will update it

File: src/app/tests/suites/certification/Test_TC_RH_2_2.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_RH_2_2.yaml:62]: i will update it

File: src/app/tests/suites/certification/Test_TC_SC_4_2.yaml (hunk 4 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_SC_4_2.yaml:101]: i will update it

File: src/app/tests/suites/certification/Test_TC_SC_4_2.yaml (hunk 3 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_SC_4_2.yaml:87]: @bzbarsky-apple forget to remove the comment i will update it

**Review by krypton36 — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
Fast-tracking test additions.
